src/instruments/src/PFD/AltitudeIndicator.tsx

```diff
@@ -209,10 +209,10 @@ export class AltitudeIndicatorOfftape extends DisplayComponent<AltitudeIndicator
                     <text id="AltFailText" class="Blink9Seconds FontLargest Red EndAlign" x="131.16769" y="83.433167">ALT</text>
                 </g>
                 <g ref={this.tcasFailed} style="display: none">
-                    <text class="Blink9Seconds FontLargest Amber EndAlign" x="141.5" y="96">T</text>
-                    <text class="Blink9Seconds FontLargest Amber EndAlign" x="141.5" y="104">C</text>
-                    <text class="Blink9Seconds FontLargest Amber EndAlign" x="141.5" y="112">A</text>
-                    <text class="Blink9Seconds FontLargest Amber EndAlign" x="141.5" y="120">S</text>
+                    <text class="Blink9Seconds FontLarge Amber EndAlign" x="141.5" y="100">T</text>
+                    <text class="Blink9Seconds FontLarge Amber EndAlign" x="141.5" y="105.5">C</text>
+                    <text class="Blink9Seconds FontLarge Amber EndAlign" x="141.5" y="110.75">A</text>
+                    <text class="Blink9Seconds FontLarge Amber EndAlign" x="141.5" y="116">S</text>
```

Comment:
```suggestion
                    <text class="Blink9Seconds FontMedium Amber EndAlign" x="141.5" y="100">T</text>
                    <text class="Blink9Seconds FontMedium Amber EndAlign" x="141.5" y="105">C</text>
                    <text class="Blink9Seconds FontMedium Amber EndAlign" x="141.5" y="110">A</text>
                    <text class="Blink9Seconds FontMedium Amber EndAlign" x="141.5" y="115">S</text>
```
This fits a little better IMO: 
![Unbenannt](https://user-images.githubusercontent.com/39596827/186249910-68b988c2-db02-452b-a01b-1b6d8320dde2.PNG)